x/emissions/proto/emissions/v1/query.proto

```diff
@@ -912,6 +914,12 @@ message QueryTopicFeeRevenueResponse {
   ];
 }
 
+message QueryChurnableTopicsRequest {}
+
+message QueryChurnableTopicsResponse {
```

Comment:
if we delete the query entirely, must the proto still stay?

Reply:
No, it can be removed. 

Question is, will you break anything if you remove the query?

Reply:
Done

Reply:
same as above just reposting for completeness:
with the "b7s removal" project we're updating the only "allora official" client, and it's prehistory in our ecosystem's history so not too worried about alternative community clients. valid concern otherwise